package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "portable-mlir-toolchain",
```

Comment:
do we want to call this `setup-mlir`?
would be kind of fitting and idiomatic. 
Could also be a good name for the entire repository/project.